setup.py

```diff
@@ -55,12 +55,19 @@ def _package_doc(name):
         'six',
         'Zope2 >= 4.0a6',
         'five.localsitemanager',
+        'AccessControl >= 4.0a4',
```

Comment:
This pin doesn't seem related to the PR.

Reply:
Sadly it is required, without this pin `tox -edocs` ends in

`ContextualVersionConflict: (AccessControl 3.0.13 (/.../Products.GenericSetup/.tox/docs/lib/python2.7/site-packages), Requirement.parse('AccessControl>=4.0a4'), set(['Products.ZCatalog']))`

Interestingly this does not happen when running the tests.

But there is a way around this: Using `pip_pre = True`. Are you fine with it?